docs/services.mdx

```diff
@@ -33,4 +33,5 @@ route: /services
 ## Wallets
 
 - [MetaMask](https://metamask.io)
-- [Hedgehog](https://hedgehog.audius.co)
\ No newline at end of file
+- [Hedgehog](https://hedgehog.audius.co)
+- [Fortmatic] (https://fortmatic.com/)
```

Comment:
Will that space be an issue for the link?

Reply:
Eagle eyes. Yes, the space is an issue.